TYPO3.Flow/Classes/TYPO3/Flow/Configuration/ConfigurationManager.php

```diff
@@ -526,6 +527,7 @@ protected function loadConfiguration($configurationType, array $packages)
                 $this->configurations[$configurationType] = array_merge($this->configurations[$configurationType], $this->configurationSource->load(FLOW_PATH_CONFIGURATION . $configurationType));
 
                 // Merge routes with SubRoutes recursively
```

Comment:
I added it there by intention because `includeSubRoutesFromSettings()` also just loads sub routes, but I'll rephrase it a bit to make that clearer